kovri.md

```diff
@@ -18,24 +18,17 @@ Kovri uses both encryption and sophisticated routing techniques to create a priv
 This protected overlay allows users to hide their geographical location and IP address.
 An IP address is like a home address for the Internet, so this is quite sensitive information, to say the least.
 
-
 ## 4) Examples
 
-Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
+Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen a user's privacy.
 
 Suppose Alice wants to send Monero to Bob.
-Alice's wallet creates a transaction, and then asks her node to broadcast it to the Monero network.
-Alice's node is connected to several nodes run by network peers, so those peers are first to receive a broadcast of her transaction, which then rebroadcast her transaction to their peers, etc.
-Nodes communicate by directing messages to their peers using the peers' IP addresses.
-This means that it is possible to geographically trace data as it travels the internet, from start to finish and everywhere in between.
-
-It is therefore evident that Alice is taking a risk by broadcasting her transaction.
-Some nodes may be logging IP addresses when transactions are received.
-Every one of Alice's node's network peers knows her IP address; therefore, each one of those peers has the ability to keep track of when a new transaction was first broadcast to them by Alice's node.
-
-Since Monero nodes tend to connect to only a limited number of peers, a transaction broadcast by Alice's node is not a conclusive indicator that Alice has initiated a transaction.
-And, even though the sender's and recipient's wallet addresses - as well as the amount of Monero sent - remain private, an adversary with enough resources could attempt to associate transactions with IP addresses to determine from where transactions originate.
-This could potentially lead to an adversary not relaying her transactions to the rest of the network; or - worse - arriving at her front door.
+Alice's wallet creates a transaction and then broadcasts it to the Monero network.
+The Monero network is made up of nodes that communicate with each other by directing messages using IP addresses.
+This means that it might be possible to geographically trace data as it travels over the open internet, from start to finish and everywhere in between.
+Even though the sender's and recipient's wallet addresses - as well as the amount of Monero sent - remain private, Alice is taking a risk in broadcasting her transaction as some nodes may be logging IP addresses.
+An adversary with enough resources could attempt to associate transactions with IP addresses to determine from where transactions originate.
+This could potentially lead to the adversary arriving at her front door.
```

Comment:
This is all good to me, and better than before, though I suggest changing the last line, 31, back to the line 38 that was removed.